pallets/automation-time/src/lib.rs

```diff
@@ -155,11 +157,30 @@ pub mod pallet {
 		}
 	}
 
+	/// The struct that stores data for a missed task.
+	#[derive(Debug, Eq, PartialEq, Encode, Decode, TypeInfo)]
+	#[scale_info(skip_type_params(T))]
+	pub struct MissedTask2<T: Config> {
+		owner_id: T::AccountId,
+		task_id: TaskId<T>,
+		execution_time: UnixTime,
+	}
+
+	impl<T: Config> MissedTask2<T> {
+		pub fn create_missed_task(
```

Comment:
I prefer making the attributes public since it reduces the boilerplate code in the `lib.rs` file required for testing. But happy to defer to you guys if this is conscious design decision.

Reply:
I will do this in a follow on PR. I want to handle the MissedTask and Task struct at the same time